acquia_lift.module

```diff
@@ -2853,35 +2778,16 @@ function acquia_lift_command_messagebox($message, $seconds = 0) {
 }
 
 /**
- * Returns an AJAX command to toggle the page variation element selector.
+ * Returns an AJAX command to toggle the element variation element selector.
  *
  * @param bool $start
- *   True to start the page variation mode, FALSE to stop it.
+ *   True to start the element variation mode, FALSE to stop it.
  */
-function acquia_lift_command_page_variation_toggle($start = TRUE) {
+function acquia_lift_command_element_variation_toggle($start = TRUE) {
   return array(
     'command' => 'acquia_lift_variation_toggle',
     'data' => array(
       'start' => $start,
-      'type' => 'page',
-    ),
-  );
-}
-
-/**
- * Returns an AJAX command to preview a specific page variation.
- *
- * @param string $agent_name
- *   The name of the campaign/agent to which the variation belongs.
- * @param int $variation_index
- *   The index of the variation to preview.
- */
-function acquia_lift_command_page_variation_preview($agent_name, $variation_index) {
```

Comment:
It is very strange that:
1) I was still able to create an web element variation.
2) acquia_lift_command_page_variation_preview traces only to acquia_lift.personalize.command.js acquia_lift_page_variation_preview(), and acquia_lift_page_variation_preview works on pageVariations object, which pageVariations is no longer being used anywhere else.

Can I see a screenshot or error log of some kind? I believe you but I can use a help on debugging this one.